### PR TITLE
feat: the Dokka docs

### PR DESCRIPTION
**Pull request by IgnatBeresnev**

### CODE REVIEW

File: docs/kr.tree [hunk 1]
```diff
@@ -323,6 +323,9 @@
             <toc-element id="maven.md" accepts-web-file-names="using-maven.html"/>
             <toc-element id="ant.md" accepts-web-file-names="using-ant.html"/>
         </toc-element>
+        <toc-element toc-title="Dokka">
+            <include from="kl" target="dokka" origin="dokka"/>
+        </toc-element>
```
Comment (IgnatBeresnev) [inline docs/kr.tree:328]: @sarahhaggarty could you please verify that I understood you correctly with the ToC?

Comment (sarahhaggarty) [reply docs/kr.tree:328]: @IgnatBeresnev, yes this is correct!

**Comment by nikpachoo** [comment]
@IgnatBeresnev, could you please include the Dokka Docs source as a git submodule? (use [this PR](https://github.com/JetBrains/kotlin-web-site/pull/3213/files) as a example)
Thus, we will be able to test the changes on demo stage.

**Review by sarahhaggarty — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -26,12 +26,13 @@ Note that source files for the [server-side landing page](https://kotlinlang.org
 
 #### Sources in different repositories
 
-Source files for coroutines and lincheck docs, and the language specification are stored in separate repositories:
+Source files for coroutines, lincheck and Dokka docs, and the language specification are stored in separate repositories:
```
[the remark this answers is not in the document]
Comment (IgnatBeresnev) [reply README.md]: Fixed

**Comment by IgnatBeresnev** [comment]
> could you please include the Dokka Docs source as a git submodule?

Forgot to push it, sorry. Pushed it now

**Comment by IgnatBeresnev** [comment]
Preview (can't actually check it since I don't have the VPN): https://branch-dokka-docs.kotlin-web-site.labs.jb.gg/

**Comment by IgnatBeresnev** [comment]
I've checked everything I wanted, LGTM :tada: 

It's ready to be deployed/merged @nikpachoo

**Comment by sarahhaggarty** [comment]
Hi @nikpachoo, I've noticed that line breaks inside tabs aren't respected. For example in the attached screenshot, there are no line breaks rendering between text and code blocks. Is this a CSS styling issue?
Screenshot 2023-01-17 at 10 20 08

**Comment by nikpachoo** [comment]
@sarahhaggarty, yeah, it looks like a CSS-issue in on the WebHelp side. Moreover, there is [a task](https://youtrack.jetbrains.com/issue/WH-4389) for them.

**Review by sarahhaggarty — APPROVED** [review]

**Comment by sarahhaggarty** [comment]
Thanks @nikpachoo Hopefully that can be resolved soon but it shouldn't block this PR. LGTM!

**Review by nikpachoo — APPROVED** [review]

**Pull request merged**